cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -749,6 +750,36 @@ __device__ inline dictionary32 const column_device_view::element<dictionary32>(
   return dictionary32{type_dispatcher(indices.type(), index_element_fn{}, indices, index)};
 }
 
+/**
+ * @brief Returns a `numeric::decimal32` element at the specified index for a `fixed_point` column.
+ *
```

Comment:
Should we include the null-element disclaimer?
```
 * If the element at the specified index is NULL, i.e., `is_null(element_index) == true`,
 * then any attempt to use the result will lead to undefined behavior.
```

Reply:
Done.